server/plugin/command.go

```diff
@@ -85,6 +88,99 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) []string {
+	mutedUsernameBytes, err := p.API.KVGet(userInfo.UserID + "-muted-users")
+	if err != nil {
+		return nil
+	}
+	mutedUsernames := string(mutedUsernameBytes)
+	var mutedUsers []string
+	if len(mutedUsernames) == 0 {
+		return mutedUsers
+	}
+	mutedUsers = strings.Split(mutedUsernames, ",")
+	return mutedUsers
+}
+
+func (p *Plugin) handleMuteList(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	return "Your muted usernames: " + strings.Join(mutedUsernames, ", ")
+}
+
+func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	var mutedUsers string
+	if len(mutedUsernames) > 0 {
+		// , is a character not allowed in github usernames so we can split on them
+		mutedUsers = strings.Join(mutedUsernames, ",") + "," + username
+	} else {
+		mutedUsers = username
+	}
+	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte(mutedUsers)); err != nil {
+		return "Error occurred saving list of muted users"
+	}
+	return "Your muted usernames: " + mutedUsers
+}
+
+func (p *Plugin) handleUnmute(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	userToMute := []string{username}
+	newMutedList := arrayDifference(mutedUsernames, userToMute)
+	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte(strings.Join(newMutedList, ","))); err != nil {
+		return "Error occurred unmuting users"
+	}
+	return "Your muted usernames: " + strings.Join(newMutedList, ", ")
+}
+
+func (p *Plugin) handleUnmuteAll(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte("")); err != nil {
+		return "Error occurred unmuting users"
+	}
+	return "Unmuted all users"
+}
+
+func (p *Plugin) handleMuteCommand(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	if len(parameters) == 0 {
+		return "Invalid mute command. Available commands are 'list', 'add' and 'delete'."
+	}
+
+	command := parameters[0]
+	var userToMute string
+	if len(parameters) == 1 && command != list && command != deleteAll {
+		return "Invalid mute command. " + command + " requires a username parameter"
```

Comment:
When the user inputs a invalid commend, the error message doesn't indicate that it's invalid. 
![Screenshot from 2020-09-15 09-57-09](https://user-images.githubusercontent.com/16541325/93182440-fabb5900-f739-11ea-816e-658acaaae1c7.png)


Maybe the sub command should handle what parameter are needed.